docs/cloud/dev/spryker-cloud-commerce-os/configure-deployment-pipelines/configuring-github-actions.md

```diff
@@ -8,19 +9,836 @@ redirect_from:
   - /docs/configuring-github-actions
   - /docs/en/configuring-github-actions
   - /docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/configuring-github-actions.html
+  - /docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/upcoming-release/configuring-github-actions.html
 ---
 
 This document describes how to configure continuous integration using GitHub Actions.
 
 ## GitHub Actions
 
-GitHub Actions help you automate your software development workflows in the same place you store code and collaborate on pull requests and issues. You can write individual tasks, called actions, and combine them to create a custom workflow. Workflows are custom automated processes that you can set up in your repository to build, test, package, release, or deploy any code project on GitHub.
+GitHub Actions helps you automate your software development workflows in the same place you store code and collaborate on pull requests and issues. You can write individual tasks, called actions, and combine them to create a custom workflow. Workflows are custom automated processes that you can set up in your repository to build, test, package, release, or deploy any code project on GitHub.
 
-For more information on GitHub Actions, see
+For more information on GitHub Actions, see the following documents:
 * [GitHub Actions](https://github.com/features/actions)
 * [Learn GitHub Actions](https://docs.github.com/en/actions/learn-github-actions)
 
 
+<details>
+<summary>Release 202304.0</summary>
+
+## Prerequisites
+
+In a GitHub repository, create the workflow configuration file: `.github/workflows/{project_name}.yml`.
+
+## Configuring basic validation with GitHub Actions
+
+Set up the job that performs the basic validation like the following:
+* Code style
+* Architecture
+* Security
+* Database schema
+
+
+<details open>
+<summary>Example of .github/workflows/{project_name}.yml</summary>
+
+```yaml
+...
+jobs:
+    ...
+    validation:
+        name: "CS, ArchSniffer, PHPStan, Security"
+        runs-on: ubuntu-18.04
```

Comment:
```suggestion
        runs-on: ubuntu-20.04
```

Reply:
According to our discussion updated `ubuntu` => `20.04` for all releases